app/src/main/java/org/oppia/app/drawer/NavigationDrawerItem.kt

```diff
@@ -2,18 +2,16 @@ package org.oppia.app.drawer
 
 import org.oppia.app.R
 
-/** Enum to store the menu of [NavigationDrawerFragment] and get menu by id. */
+/** Enum Represents different destinations that can be selected by the user from the navigation drawer. */
```

Comment:
`Enum represents different destinations that can be selected by the user from the navigation drawer.`

first character of represents should be lowercase.

Reply:
or remove `Enum` first word.

Reply:
Removed Enum